### ISSUE TITLE
[Questions] Does client-side entity rendering continually create new entities? How do I detect these client entities? And can an entity's health render be disabled?

### ISSUE THREAD

**Issue by The-Fireplace** [McJty/TheOneProbe#189]
From a look at your code, it looks like it caches the entity for a number of ticks or milliseconds or something, then creates a new one each time the old one "expires". My problem here that leads me to ask is what happens with my entities the first time they render. They attempt to cache some extra data, which can't be retrieved from the server for your client-only entities because they don't exist on the server. This results in a TON of server log spam, and appears to eat up a ton of entity IDs in the client world. This is the log from looking at one Mechanical Skeleton for a few seconds.
http://paste.ubuntu.com/25245059/
So, to resolve this, is there any way I can check when rendering the entity to see if it is created by The One Probe, so I can skip trying to cache the data?

Also, a side question I have, is there any way to disable rendering the health in the information? I tried using ITheOneProbe#registerEntityDisplayOverride, but even when just setting the IEntityDisplayOverride#overrideStandardInfo to return true, the health still renders.

**Comment by McJty** [comment]
Can't you simply not do that server spam then or only do it once?

**Comment by McJty** [comment]
Btw I am on vacation right now so can't do much

**Comment by The-Fireplace** [comment]
Yes, I have currently disabled that console output when TOP is loaded. Enjoy your vacation!
